amman/src/validator/types.ts

```diff
@@ -78,6 +80,7 @@ export type ValidatorConfig = {
   limitLedgerSize: number
   verifyFees: boolean
   detached: boolean
+  features: 'mainnet' | 'devnet' | 'testnet' | string[]
```

Comment:
Shouldn't this be `string` instead of `string[]` as you're expecting _one_ not _many_?

Reply:
Well the reason I did it this way was to either choose a specific feature disactivation of one of the current nets or you could pass an array of all of the feature pubkeys you want to disactivate.